run-onboarding.sh

```
@@ -287,7 +287,7 @@
     update_props
     echo "Starting online-verification-partner creation" $'\n'
     jq '.values |= map(if .key=="cert-application-id" then (.value="IDA") else . end)' onboarding.postman_environment.json > $(prop 'tmp_dir')/tmp.json && mv $(prop 'tmp_dir')/tmp.json onboarding.postman_environment.json
-    jq '.values |= map(if .key=="cert-reference-id" then (.value="'$partner_kc_username'") else . end)' onboarding.postman_environment.json > $(prop 'tmp_dir')/tmp.json && mv $(prop 'tmp_dir')/tmp.json onboarding.postman_environment.json
+    jq '.values |= map(if .key=="cert-referen(&here as well to check coderabbit)ce-id" then (.value="'$partner_kc_username'") else . end)' onboarding.postman_environment.json > $(prop 'tmp_dir')/tmp.json && mv $(prop 'tmp_dir')/tmp.json onboarding.postman_environment.json
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)